app/Services/Doctor/VaccinationSlotService.cs

```diff
@@ -31,12 +31,16 @@ public async Task<NewVaccinationSlotResponse> AddNewSlot(NewVaccinationSlotReque
                 throw new ValidationException();
             }
 
+            // Check for date in the past
+            if(date < DateTime.Now)
```

Comment:
Dodaj przypadek w testach, żeby sprawdzał to

Reply:
Zrobione 👍